app/scripts/metamask-controller.js

```diff
@@ -7770,7 +7690,7 @@ export default class MetamaskController extends EventEmitter {
       txParams: { from },
     } = transactionMeta;
     this.accountTrackerController.updateAccountByAddress({
-      from,
+      address: from,
       networkClientId,
```

Comment:
🤯 what were the implications here? Was this bug not caught by an e2e test because up until now we weren't using the state that it updated?